tracker/event_tracker_store.go

```diff
@@ -0,0 +1,331 @@
+package tracker
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/json"
+	"fmt"
+
+	hcf "github.com/hashicorp/go-hclog"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/tracker/store"
+	bolt "go.etcd.io/bbolt"
+)
+
+var _ store.Store = (*EventTrackerStore)(nil)
+
+var (
+	dbLogs          = []byte("logs")
+	dbConf          = []byte("conf")
+	dbNextToProcess = []byte("nextToProcess")
+)
+
+// EventTrackerStore is a tracker store implementation.
+type EventTrackerStore struct {
+	conn          *bolt.DB
+	finalityDepth uint64
+	notifierCh    chan<- []*ethgo.Log
+	logger        hcf.Logger
+}
+
+// NewEventTrackerStore creates a new EventTrackerStore
+func NewEventTrackerStore(
+	path string,
+	finalizedTrashhold uint64,
+	notifierCh chan<- []*ethgo.Log,
+	logger hcf.Logger) (*EventTrackerStore, error) {
+	db, err := bolt.Open(path, 0600, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	store := &EventTrackerStore{
+		conn:          db,
+		finalityDepth: finalizedTrashhold,
```

Comment:
You had a typo in `finalizedTrashhold`, so I propose to rename it to `finalityDepth` or `finalizedThreshold` (if you find it more suitable).
```suggestion
	finalityDepth uint64,
	notifierCh chan<- []*ethgo.Log,
	logger hcf.Logger) (*EventTrackerStore, error) {
	db, err := bolt.Open(path, 0600, nil)
	if err != nil {
		return nil, err
	}

	store := &EventTrackerStore{
		conn:          db,
		finalityDepth: finalityDepth,
```

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1228/commits/7c00602ddd77823dd9574710897a8307d0399e5f